ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -212,6 +219,30 @@ Verify that the must-gather image provides RHODS logs and info
     [Teardown]  Cleanup must-gather Logs
 
 
+Verify That DSC And DSCI Release.Name Attribute matches ${expected_release_name}
+    [Documentation]    Tests the release.name attribute from the DSC and DSCI matches the desired value.
+    ...                ODH: Open Data Hub
+    ...                RHOAI managed: OpenShift AI Cloud Service
+    ...                RHOAI selfmanaged: OpenShift AI Self-Managed
+    [Tags]    Upgrade
+    Should Be Equal As Strings    ${DSC_RELEASE_NAME}     ${expected_release_name}
+    Should Be Equal As Strings    ${DSCI_RELEASE_NAME}    ${expected_release_name}
```

Comment:
Can't we reuse the same test from the previous file and just include it in the tag Upgrade? Both tests do the same so why do we have to have them duplicated?

If for some reason they have to be duplicated, can they have exactly the same steps? Or is there any reason why for upgrade tests we get the release name and version in the test suite setup and not in the tests themselves like we do it in the `operator.robot` file?